fcm-client.cabal

```diff
@@ -18,6 +18,7 @@ tested-with:         GHC == 8.4.4
                      GHC == 8.8.4
                      GHC == 8.10.4
                      GHC == 8.10.7
+                     GHC == 9.0.2
```

Comment:
I'd remove the 8.10.4 from the "tested-with" now that we're dropping that from CI

Reply:
Done.